src/main/java/org/terasology/logic/health/event/DamageSoundComponent.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.logic.health.event;
+
+import com.google.common.collect.Lists;
+import org.terasology.audio.StaticSound;
+import org.terasology.entitySystem.Component;
+
+import java.util.List;
+
+/**
```

Comment:
Blank javadoc - leftovers from when we did a bulk remove of `@author` tags ages ago. Feel free to remove them (or better yet fill them in!) wherever you see them